.github/workflows/circleci.yml

```diff
@@ -13,6 +13,7 @@ jobs:
         uses: larsoner/circleci-artifacts-redirector-action@master
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+          api-token: ${{ secrets.CIRCLE_TOKEN }}
```

Comment:
```suggestion
          api-token: ${{ secrets.CIRCLECI_TOKEN }}
```

There was an inconsistency in the naming in the instructions, I copied the name from step 3 you copied the name from step 2!

Reply:
![image](https://user-images.githubusercontent.com/199813/231561570-6e41bb24-8242-4ebd-aea8-29b5d6771739.png)

Reply:
I can not rename it without fully regenerating it 😞